docs/_renderer.py

```diff
@@ -158,15 +159,15 @@ def render(self, el: ds.DocstringParameter):
         # Wrap everything in a code block to allow for links
         param = "<code>" + param + "</code>"
 
-        clean_desc = sanitize(el.description, allow_markdown=True)
+        clean_desc = el.description.replace("|", "&vert;")
         return (param, clean_desc)
```

Comment:
Definitions in the definition list don't require sanitization, markdown allowed! (We do indent new lines later when creating the definition list markup.)

```suggestion
        return (param, el.description)
```